src/routes/forgot-password/components/ForgotPassword.js

```diff
@@ -0,0 +1,106 @@
+const React = require('react');
+const T = require('prop-types');
+const StrangeForms = require('strange-forms');
+const IsEmail = require('utils/is-email');
+
+module.exports = class ForgotPassword extends StrangeForms(React.Component) {
+
+    static propTypes = {
+        requestReset: T.func.isRequired,
+        errorMessage: T.string
+    };
+
+    constructor(props) {
+
+        super(props);
+
+        this.state = {
+            email: '',
+            isBlurred: {
+                email: false
+            }
+        };
+
+        this.strangeForm({
```

Comment:
This was my direction, so it's my fault, but strangeforms could be overkill for a simple form like this.  Curious what others think.

Reply:
I don't think it's overkill :)

Reply:
There have been some forms that I skipped using strangeforms, and regretted it as the form grew.  But also, once you are familiar with its options, it saves some time even for very small forms.